### PR TITLE
Add virtio-rng-pci params

### PR DESCRIPTION
**Pull request by Jason-ZW**
Add virtio-rng-pci params

### CODE REVIEW

File: scripts/run [hunk 1]
```diff
@@ -278,7 +278,8 @@ if [ "$QIND" != "1" ]; then
             ;;
             *)
                 CLOUD_CONFIG_DISK="-fsdev local,security_model=passthrough,readonly,id=fsdev0,path=${CCROOT} \
-                    -device virtio-9p-pci,id=fs0,fsdev=fsdev0,mount_tag=config-2"
+                    -device virtio-9p-pci,id=fs0,fsdev=fsdev0,mount_tag=config-2 \
+                    -device virtio-rng-pci"
```
Comment (niusmallnan) [inline scripts/run]: redundant

File: scripts/run [hunk 2]
```diff
@@ -330,6 +331,7 @@ if [ "$QEMU" == "1" ]; then
             ${CLOUD_CONFIG_DISK} \
             -fsdev local,security_model=none,id=fsdev1,path=${HOME} \
             -device virtio-9p-pci,id=fs1,fsdev=fsdev1,mount_tag=home \
+            -device virtio-rng-pci \
```
Comment (niusmallnan) [inline scripts/run]: redundant

File: scripts/run [hunk 3]
```diff
@@ -338,7 +340,8 @@ elif [ "$BOOT_ISO" == "1" ] ||
     if [ "$BOOT_ISO" == "1" ]; then
         ISO_OPTS="-boot d -cdrom ./dist/artifacts/rancheros.iso \
             -fsdev local,security_model=none,id=fsdev1,path=${HOME} \
-            -device virtio-9p-pci,id=fs1,fsdev=fsdev1,mount_tag=home "
+            -device virtio-9p-pci,id=fs1,fsdev=fsdev1,mount_tag=home \
+            -device virtio-rng-pci "
```
Comment (niusmallnan) [inline scripts/run]: redundant

Comment (Jason-ZW) [reply scripts/run]: Already modified

File: scripts/run (hunk 2 above)
Comment (Jason-ZW) [reply scripts/run]: Already modified

File: scripts/run (hunk 1 above)
Comment (Jason-ZW) [reply scripts/run]: Already modified

**Pull request merged**